src/bundle/Resources/views/themes/admin/content/edit_base.html.twig

```diff
@@ -34,47 +58,21 @@
                         <div class="ibexa-anchor-navigation-sections">
                             {% for key, group in grouped_fields %}
                                 {% embed '@ibexadesign/ui/anchor_navigation_section.html.twig' with {
-                                    anchor_section_key: key
+                                    anchor_section_key: key,
+                                    parentSelf: _self,
                                 } %}
                                     {% trans_default_domain 'content_edit' %}
+                                    {% import parentSelf as parentSelf %}
```

Comment:
Note: it seems that it needs to be done this way.
ref. https://github.com/twigphp/Twig/issues/2442#issuecomment-968096087